main.tf

```diff
@@ -1,13 +1,98 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+#-----------------------------------------------------------------------------------------------------------------------
+# Setup AWS Inspector
+#-----------------------------------------------------------------------------------------------------------------------
+module "insepector_assessment_target_label" {
+  source  = "cloudposse/label/null"
+  version = "0.21.0"
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+  attributes = concat(module.this.attributes, ["inspector", "assessment", "target"])
+  context    = module.this.context
+}
+
+resource "aws_inspector_assessment_target" "target" {
+  name = module.insepector_assessment_target_label.id
+}
+
+module "insepector_assessment_template_label" {
+  source  = "cloudposse/label/null"
+  version = "0.21.0"
+
+  attributes = concat(module.this.attributes, ["inspector", "assessment", "template"])
+  context    = module.this.context
+}
+
+resource "aws_inspector_assessment_template" "assessment" {
+  name               = module.insepector_assessment_template_label.id
+  target_arn         = aws_inspector_assessment_target.target.arn
+  duration           = var.assessment_duration
+  rules_package_arns = var.enabled_rules
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Create a scheduled event to run inspector
+#-----------------------------------------------------------------------------------------------------------------------
+module "insepector_schedule_label" {
+  source  = "cloudposse/label/null"
+  version = "0.21.0"
+
+  attributes = concat(module.this.attributes, ["inspector", "schedule"])
+  context    = module.this.context
+}
+
+resource "aws_cloudwatch_event_rule" "schedule" {
+  count               = local.create_scheduled_event ? 1 : 0
+  name                = module.insepector_schedule_label.id
+  description         = "Trigger an AWS Inspector Assessment"
```

Comment:
lets parameterize in a var, with this as the defautl

Reply:
7df408c6f4ebaffe167ee3d0f3a776d0dd644845